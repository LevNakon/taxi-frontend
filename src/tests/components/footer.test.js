import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Footer from '../../components/footer';

describe('Footer --- test render correctly component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Footer />);
    });

    it('render correctly component Footer', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

    it('div with class footer and p exist in component Footer',()=>{
        expect(wrapper.find('div')).toMatchSelector('.footer');
        expect(wrapper.find('p')).toExist();
    });
});