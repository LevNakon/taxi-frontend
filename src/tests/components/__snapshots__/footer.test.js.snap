// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer --- test render correctly component render correctly component Footer 1`] = `
<div
  className="footer"
>
  <p>
    Created by Lev Nakonechnyy
  </p>
</div>
`;
